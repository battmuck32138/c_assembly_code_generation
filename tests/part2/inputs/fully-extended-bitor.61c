int main () {
  return (1 | 6) | (8 | 32);  
} 
