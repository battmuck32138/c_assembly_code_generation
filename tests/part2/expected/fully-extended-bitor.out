
Exited with error code 47
